File: ex09/ex09_02/ex09_02.js
function main() {
  const persons = new Map([
    ["Chang", 45],
    ["Chen", 36],
    ["Yu", 39],
    ["Wang", 25]
  ]);
    
  let it = persons.entries();
  for (let item of it) {
    console.log(item[1]);
  }
  return 0;
}
main();

File: ex09/ex09_02/ex09_02.cpp
#include <iostream>
#include <map>
using namespace std;

int main() {
  map<string, int> persons = {
    {"Chang", 45},
    {"Chen", 36},
    {"Yu", 39},
    {"Wang", 25}
  };
  map<string, int>::iterator it;
  for (it = persons.begin(); it != persons.end(); ++it) {
      cout << it->second << endl;
  }
  return 0;
}